python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -404,9 +488,9 @@ async def do_applyCorrection(self, id_data):
         #                      self.set_pressure("m2", azimuth, elevation),
         #                      )  # FIXME: What about focus? YES, do focus separately
 
-    async def do_applyFocusOffset(self, id_data):
-        """Adds a focus offset to the focus correction. Same as apply focus
-         but do the math...
+    async def do_applyFocusOffset(self, id_data, user_supplied=True):
```

Comment:
These commands should not have additional arguments. They will be called by the high level class in response to commands. If you need a `user_supplied` flag, you should probably use a class attribute or add it to the topic.

Reply:
ok, This can be removed so long as no internal function is allowed to call it.
Note that if an internal function were to call it, then it'll cause accounting errors for the focus contributions.

Reply:
What do you mean by "an internal function were to call it"? This method will be called as a response to the `applyFocusOffset` command. Which, in a way, is an "internal call".

Reply:
What exactly you where trying to accomplish with this parameter? it seems like you may be trying to implement some kind of locking mechanism.

Reply:
Added docstring warnings as requested.